pkg/apiserver/experiment/experiment.go

```diff
@@ -47,12 +51,13 @@ func NewService(
 
 // Register mounts our HTTP handler on the mux.
 func Register(r *gin.RouterGroup, s *Service) {
-	endpoint := r.Group("/experiment")
+	endpoint := r.Group("/experiments")
 
 	// TODO: add more api handlers
 	endpoint.GET("/", s.listExperiments)
-	endpoint.GET("/:name", s.getExperimentDetail)
+	endpoint.GET("/detail/:name", s.getExperimentDetail)
```

Comment:
IMO this API is not RESTful. And do we also need `ns` in this API?

Reply:
I think ns is needed, and this can be added when the API is developed.